mfhdf/test/tncvargetfill.c

```diff
@@ -60,6 +60,89 @@
 #define SDS2_NAME  "Variable 2"
 #define D1_NUMRECS 8
 #define D2_NUMRECS 2
+
+/*
+ * Helper function: Read and verify data of the variables with nc API
+ */
```

Comment:
I separated a couple of functions out to avoid the shadowing warnings when the copy-pasted variables in the blocks shadow the variables in the function body.